rbac/rbac/middleware.py

```diff
@@ -123,11 +120,12 @@ def _create_tenant(account):
         schema_name = create_schema_name(account)
         try:
             with transaction.atomic():
-                tenant = Tenant(schema_name=schema_name)
-                tenant.save()
-                logger.info('Created new tenant from account_id %s.', account)
-                seed_roles(tenant=tenant, update=False)
-                seed_group(tenant=tenant)
+                tenant, created = Tenant.objects.get_or_create(schema_name=schema_name)
```

Comment:
Noob question: does get_or_create raise any kind of errors? This is more for my knowledge than an actual critique of any kind.

Reply:
I only know that if the key specified is not unique, e.g., several tenants with schema_name 'acct1234' exist, it will return MultipleObjectsReturned. But this would not happen in our case since this is the primary key for tenant.